### PR TITLE
Docs 9221 evaluate and remove versions from mule links in rtm hn

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Review by MichelleZMY — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/developing-applications-for-cloudhub.adoc [hunk 1]
```diff
@@ -32,59 +35,73 @@ http://0.0.0.0:${http.port}/service/path/two
...
+The CloudHub logging service captures all log messages from your application.
+You can view logs in the *Logging* tab in the Runtime Manager console.
+To expose additional information in your logs for diagnostic or audit purposes,
+use the xref:mule-runtime::logger-component-reference.adoc[Logger component].
 
 [IMPORTANT]
-On CloudHub, after you delete an application, your log data is no longer accessible through the console. CloudHub archives old log data for a limited period of time before being purged. This allows you to recover the data if needed. Open a support case at: https://help.mulesoft.com[MuleSoft Help Center] for more information.
+On CloudHub, after you delete an application, your log data is no longer accessible through the console. CloudHub archives old log data for a limited period of time before being purged
```
Comment (valkyrie69) [inline modules/ROOT/pages/developing-applications-for-cloudhub.adoc]:
```suggestion
On CloudHub, after you delete an application, your log data is no longer accessible through the console. CloudHub archives old log data for a limited period of time before it is purged, 
```

File: modules/ROOT/pages/developing-applications-for-cloudhub.adoc [hunk 2]
```diff
@@ -32,59 +35,73 @@ http://0.0.0.0:${http.port}/service/path/two
...
+On CloudHub, after you delete an application, your log data is no longer accessible through the console. CloudHub archives old log data for a limited period of time before being purged 
+so you can recover the data if needed.
+For more information, open a support case at https://help.mulesoft.com[MuleSoft Help Center].
 
 == Package Third-Party Libraries
 
-CloudHub provides services in the platform that support connecting over many different protocols and transforming many data formats. All the standard Mule libraries are included and available to your application. If you have the need to include a different third party library, you can package that library in the `<mule application zip>/lib` folder of your xref:3.8@mule-runtime::application-format.adoc[application package]. Any libraries that are packaged in your Mule application `lib` folder are available on the classpath to your application at run time.
+CloudHub provides services in the platform that support connecting over different protocols
```
Comment (valkyrie69) [inline modules/ROOT/pages/developing-applications-for-cloudhub.adoc]:
```suggestion
CloudHub provides services in Anypoint Platform that support connecting over different protocols
```

File: modules/ROOT/pages/managing-application-data-with-object-stores.adoc [hunk 3]
```diff
@@ -9,7 +9,10 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-CloudHub includes application data storage for each application it manages, this data is accessible through Mule's xref:3.8@mule-runtime::mule-object-stores.adoc[Object Store v1] and xref:object-store::index.adoc[Object Store v2] functionality. Object Stores can be used to store anything, but two of its main uses are:
+CloudHub includes application data storage for each application it manages,
```
Comment (valkyrie69) [inline modules/ROOT/pages/managing-application-data-with-object-stores.adoc]:
```suggestion
CloudHub includes application data storage for each application it manages.
```

File: modules/ROOT/pages/managing-application-data-with-object-stores.adoc [hunk 4]
```diff
@@ -9,7 +9,10 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
...
+this data is accessible through the Mule object store and Object Store v2 functionality.
```
Comment (valkyrie69) [inline modules/ROOT/pages/managing-application-data-with-object-stores.adoc]:
```suggestion
This data is accessible through the Mule object store and Object Store v2 functionality.
```

File: modules/ROOT/pages/managing-application-data-with-object-stores.adoc [hunk 5]
```diff
@@ -9,7 +9,10 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
...
+
+Object Stores can be used to store anything, but two of its main uses are:
```
Comment (valkyrie69) [inline modules/ROOT/pages/managing-application-data-with-object-stores.adoc]: Is it Object stores?

File: modules/ROOT/pages/managing-queues.adoc [hunk 6]
```diff
@@ -38,7 +38,9 @@ The *Queues* tab is useful for the following use cases:
 
 To enable queues:
 
-.  Deploy an application to CloudHub that includes one or more xref:3.8@mule-runtime::vm-transport-reference.adoc[VM endpoints] or that includes a flow set to the queued-asynchronous xref:3.8@mule-runtime::flow-processing-strategies.adoc[processing strategy].
+. Deploy an application to CloudHub that includes one or more queues. 
++
+See xref:connectors::vm/vm-connector.adoc[VM Connector].
 .  In the Runtime Manager, click the application and click *Manage Application*.
```
Comment (valkyrie69) [inline modules/ROOT/pages/managing-queues.adoc]:
```suggestion
.  In Runtime Manager, click the application and click *Manage Application*.
```

File: modules/ROOT/pages/managing-queues.adoc [hunk 7]
```diff
@@ -38,7 +38,9 @@ The *Queues* tab is useful for the following use cases:
...
 .  Click the checkbox for *Persistent queues*.
 .  If you are enabling this setting when deploying an application, click *Deploy Application*. If you are updating an existing application, click *Apply Changes*.
```
Comment (valkyrie69) [inline modules/ROOT/pages/managing-queues.adoc]:
```suggestion
.  If you are enabling this setting when deploying an application, click *Deploy Application*. +
If you are updating an existing application, click *Apply Changes*.
```

**Review by valkyrie69 — DISMISSED** [review]

File: modules/ROOT/pages/managing-application-data-with-object-stores.adoc (hunk 5 above)
Comment (hannanelson) [reply modules/ROOT/pages/managing-application-data-with-object-stores.adoc]: Yep. Fixed.

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**